Makefile

```diff
@@ -8,6 +8,10 @@ sagefile := .sage/bin/sagefile
 $(sagefile): .sage/go.mod $(shell find .sage/.. -type f -name '*.go')
 	@cd .sage && go mod tidy && go run .
 
+.PHONY: build-sage
```

Comment:
I have a few thoughts here:

1) Building the sagefile binary is so quick now that we have no dependencies that I think we should make it a PHONY target and always rebuild it
2) I think that this target should simply be an alias for the $(sagefile) target, since they do the same thing

Reply:
@viktorvoltaire could you chime in here?

Reply:
I like how you are thinking @odsod just tested it now and building takes half a second :) lets go for it! @Holmus sure I can push a fix

Reply:
Fixed! @Holmus @odsod @Edholm  what do think of the name `build-sage` ? another option could just be `make sage` :)

Reply:
I kinda like `make sage`

Reply:
:heavy_plus_sign: on `make sage`

Reply:
sage it is!